docs/source/interactive/python-ipython-diff.rst

```diff
@@ -126,8 +126,8 @@ and if possible display the python source code of this object.
 
 
 
-If you are looking for an object, the use of wildcards ``*`` in conjunction
```

Comment:
I think temoval of this comma makes the sentence nonsensical? What's wrong here?